recipe/meta.yaml

```diff
@@ -5,30 +5,32 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/d/descarteslabs/descarteslabs-{{ version }}.tar.gz
-  sha256: 720ecf89f2f2347d0aabaf3ca5b74b53fb93f55278c11661a47dc23203db60c4
+  url: https://github.com/descarteslabs/descarteslabs-python/archive/v{{ version }}.tar.gz
+  sha256: 29eb6f6579f2d772e058382b667d5506012f235e6c892277b8572909d5a845d4
 
 build:
-  number: 0
-  noarch: python
+  number: 1
+  skip: True  # [py<36]
   script: {{ PYTHON }} -m pip install . --no-deps -vv
   entry_points:
     - descarteslabs = descarteslabs.client.scripts.__main__:main
 
 requirements:
   build:
-    - python >=3.5
+    - python
     - pip
   run:
-    - python >=3.5
+    - python
     - affine >=2.2.1
+    - backports-datetime-fromisoformat >=1.0.0  # [py<37]
     - cachetools >=2.0.1
-    - cloudpickle ==0.4.0
+    - cloudpickle ==0.4.0  # [py<38]
+    - cloudpickle ==1.6.0  # [py>=38]
     - geojson >=2.5.0
     - grpcio >=1.16.1,<2
-    - protobuf ==3.11.2,<4
+    - protobuf >=3.11.2,<4
     - numpy >=1.17.4
-    - pyarrow
+    - pyarrow >=0.17.1,<1
```

Comment:
This one should be ==0.17.1 but I;m hoping they where conservative in the metadata and hopefully the tests will catch any problem.